.github/workflows/build.yml

```diff
@@ -0,0 +1,102 @@
+name: Build and test
+
+on: [push]
+
+jobs:
+  build:
+    name: Build and test for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-2016, macOS-latest]
+    steps:
+      - name: Enable symlinks
+        run: git config --global core.symlinks true
+
+      - name: Checkout the repository
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+          submodules: true
+
+      - name: install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+
+      - name: install rust
+        uses: hecrj/setup-rust-action@v1
+        with:
+          rust-version: "1.37.0"
+
+      - name: install clippy and rustfmt
+        run: |
+          rustup component add clippy
+          rustup component add rustfmt
+
+      - name: install python
+        uses: actions/setup-python@v1
+        with:
+          python-version: "2.7.16"
+          architecture: x64
+
+      - name: Setup python package dir
+        if: startsWith(matrix.os, 'windows')
+        run: setx PYTHONPATH "third_party\python_packages" /m
+
+      - name: print versions
+        run: |
+          node -v
+          python --version
+          rustc --version
+          cargo --version
+
+      - name: setup env
+        run: |
+          echo ::set-env name=RUSTC_WRAPPER::sccache
+          echo ::set-env name=DENO_BUILD_MODE::release
+          echo ::add-path::prebuilt/win
+          echo ::add-path::`pwd`/prebuilt/mac
+          echo ::add-path::`pwd`/prebuilt/linux64
+
+      - name: tools/setup.py
+        env:
+          GYP_MSVS_OVERRIDE_PATH: C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise
```

Comment:
It feels very wrong that this would be needed. What happens if you don't do it?

Reply:
It tries to use vs community, but that doesn't exist on the runner so building fails.

Reply:
I have exactly the same version of Visual Studio on my machine, and I don't need to set that env var.